src/shared/components/post/post-listing.tsx

```diff
@@ -591,7 +591,10 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
   }
 
   duplicatesLine() {
-    const dupes = this.props.crossPosts;
+    const dupes = this.props.crossPosts?.filter(
+      pv => pv.community.id !== this.postView.community.id
```

Comment:
Using the `pv.post.id` is better here, because a single post could have some historical cross-posts. For example, a github project might be posted multiple times over the course of a few years.